### DISCUSSION TITLE
Using UABEANext, strange occurence with decompiling a file in a game

### DISCUSSION THREAD

**Discussion by prinpop** [nesrak1/UABEA#323] [Q&A]
Hello! Apologizes if this isn't the right place for this, I'm really new to this (I also may not know any sort of proper/technical jargon...). I even made a GitHub account to ask this question!

I used UABEANext to find dialog/text for a game made in Unity. However, I noticed something a bit strange when I exported it as JSON file, where the output was this:
![image](https://github.com/nesrak1/UABEA/assets/167223851/69a1a4b6-2b46-4334-8bee-e4f95d55142f)
It didn't have any of the text at all, so I thought I made a mistake initially... Until I checked it out on AssetStudio, where I was able to see some of the dialog I was looking for, albeit out of order. When I exported it from AssetStudio, though, it looked a bit different than the preview gave. That made me a bit curious, so I decided to check UABEA again and decided to export it as a text file instead of a JSON, and the output changed:
![image](https://github.com/nesrak1/UABEA/assets/167223851/61d5ae62-6a9a-483c-97f2-51d45e0b8985)
Unfortunately, the text is still out of order(?) and filled with gibberish (I believe some of the text might be in Korean / Japanese / TCN / SCN, but got corrupted somehow? it's strange because I haven't seen UABEA struggle with those languages before). It made me wonder why this occurred in the first place, and also makes me wonder if I can maybe export it differently to get less garbled results.

**Comment by nesrak1** [comment]
The output is correct for json export. If you want the binary of this text asset, you should use the TextAsset export plugin. The version of UABEANext currently doesn't have plugins, so you would have to do this in UABEA.

**Reply by prinpop** [reply]
Thank you for replying! Unfortunately the text still comes out corrupted (unless the text was in English) and I can't get the other language text to show up properly... But I believe this is an issue with the game itself, not UABEA, as I have checked other games (mainly JP) and the output was fine.

**Reply by nesrak1** [reply]
I don't have the data, but it just looks encoded (like utf-16 or something). Either that, or the data is some other binary, not CJK text.
